File: POO/PHP/UberX.php
<?php 
require_once('Car.php');

class UberX extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver, $brand, $model);

        $this->brand = $brand;
        $this->model = $model;
    }
    public function setPassenger($passenger){
        if($passenger == 4){
            $this->passenger = $passenger;
        }else{
            echo("UberX necesita 4 pasajeros");
        }
    }

}

File: POO/Java/UberX.java
class UberX extends Car{
    String brand;
    String model;
    
    public UberX(String license, Account driver, String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }

    public void setPassenger(Integer passenger){
        if(passenger == 4){
            super.setPassenger(passenger);
        }else{
            System.out.println("UberX necesita 4 pasajeros");
        }
    }
}